Make.config

```diff
@@ -36,14 +37,17 @@ else
 CURRENT_BRANCH:=$(PACKAGE_HEAD_BRANCH)
 endif
 
+# for service releases and previews / shared for XI and XM packages
+PACKAGE_VERSION_REV=8
+
 IOS_PRODUCT=Xamarin.iOS
 IOS_PACKAGE_NAME=Xamarin.iOS
 IOS_PACKAGE_NAME_LOWER=$(shell echo $(IOS_PACKAGE_NAME) | tr "[:upper:]" "[:lower:]")
 # NEVER customize IOS_PACKAGE_VERSION itself, other parts (mtouch, web updater) are using the IOS_PACKAGE_VERSION_* variables
-IOS_PACKAGE_VERSION=10.99.7.$(IOS_COMMIT_DISTANCE)
+IOS_PACKAGE_VERSION=10.99.$(PACKAGE_REVISION).$(IOS_COMMIT_DISTANCE)
```

Comment:
You're using `PACKAGE_REVISION` (instead of `PACKAGE_VERSION_REF` like elsewhere) here.

Reply:
nice catch, that was the original name :)